cmd/notary-server/main.go

```diff
@@ -170,6 +176,36 @@ func getTrustService(configuration *viper.Viper, sFactory signerFactory,
 	return notarySigner, keyAlgo, nil
 }
 
+// Gets the cache configuration for GET-ting metadata.  This is the max-age
+// (an integer in seconds, just like in the Cache-Control header) for consistent
+// (content-addressable) downloads and current (latest version) downloads.
+// The max-age must be between 0 and 31536000 (one year in seconds, which is
+// the recommended maximum time data is cached), else parsing will return an
+// error.  A max-age of 0 will disable caching for that type of download
+// (consistent or current).
+func getCacheConfig(configuration *viper.Viper) (*handlers.CacheControlConfig, error) {
```

Comment:
Thinking maybe this and the other config slicing functions should maybe be moved in a `cmd/notary-server/(config|setup).go` file? There's ~150 LOC which seems like a sufficient amount to justify its own file rather than creating noise with the `main()` in here.